app/views/provider_interface/start_page/show.html.erb

```diff
@@ -25,6 +25,16 @@
   </div>
 </div>
 
+<% if FeatureFlag.active?('covid_19') %>
```

Comment:
@willmcb Two small things:

* I don’t think we need to duplicate `govuk-width-container`; the banner can be placed within and at the top of the existing container
* Is there not an existing view component that can be used instead of raw HTML?

Reply:
Not that I know of but there may be. I'll ask around.